__tests__/mockDsOs/example_data_source.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
```

Comment:
change folder `mockDsOs` to `mock_ds_os`

Reply:
done